src/photos/components/Viewer.jsx

```diff
@@ -199,13 +202,13 @@ export class Viewer extends Component {
         x: 0,
         y: 0
       },
-      hideToolbar: true
+      hideActions: true
     }))
   }
 
   componentDidUpdate (prevProps, prevState) {
-    // if the toolbar was hidden but is now displayed, start a countdown to hide it again
-    if (prevState.hideToolbar && !this.state.hideToolbar) this.hideToolbarAfterDelay()
+    // if the viewer's actions were hidden but are now displayed, start a countdown to hide them again
```

Comment:
If I understand this comment:

1. viewer's actions were hidden
1. but are now displayed
1. start a countdown to hide them again

Let's see the code (next comment)

Reply:
I just updated the comment to be honest.